playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -133,25 +134,29 @@ def report_pod_issue(
     # find pods with issues
     pods_with_issue = [pod for pod in pods if detect_pod_issue(pod) == issue]
     pod_names = [pod.metadata.name for pod in pods_with_issue]
+
+    if len(pods_with_issue) < 1:
+        logging.debug(f"`pods_with_issue` for found for issue: {issue}")
+        return
+
     expected_pods = get_expected_replicas(event)
```

Comment:
I think this needs to change
We have here the `event`, the type of the issue, the wait message and wait reason (which are currently not in use)

We need a new function, `get_explanation`
Information about number of available pods, and number of unavailable should be taken from the resource status

For Deployment, Statefulset, DaemonSet, message should be:
X pods are available, Y pods are not ready due to $ISSUE TYPE.
($WAIT_REASON: $WAIT_MESSAGE)

For others (Pod, Jobs ...)
Pod is in not ready due to $ISSUE_TYPE

Reply:
done